multiload/main.c

```diff
@@ -288,23 +288,14 @@ multiload_applet_tooltip_update(LoadGraph *g)
 		g_assert_not_reached();
 
 	if (!strncmp(g->name, "memload", strlen("memload"))) {
-		guint mem_user, mem_cache, user_percent, cache_percent;
-		mem_user  = g->data[1][0];
-		mem_cache = g->data[1][1] + g->data[1][2] + g->data[1][3];
-		user_percent = 100.0f * mem_user / g->draw_height;
-		cache_percent = 100.0f * mem_cache / g->draw_height;
-		user_percent = MIN(user_percent, 100);
-		cache_percent = MIN(cache_percent, 100);
-
-		/* xgettext: use and cache are > 1 most of the time,
-		   please assume that they always are.
-		 */
+		guint percent;
+		percent = 100.0f * (g->data[1][0] + g->data[1][1]) / g->draw_height;
+		percent = MIN(percent, 100);
```

Comment:
The percentage is based on the number of pixels (the height of the graph), error = +/- 100/height (+/-100/24 =+/-5% ). I recommend to use a variable to store the actual value for displaying this percentage.

Reply:
I think you did something better in your PR, by using some float value defined elsewhere instead, but I did not understand it enough to replicate it. So I tried to keep it kind of close to what it was, and assumed you will apply your idea on top of this along with other fixes in your PR.

Reply:
See `percentage_used` on #467